src/v/kafka/server/tests/alter_config_test.cc

```diff
@@ -1214,3 +1218,337 @@ FIXTURE_TEST(
     assert_property_value(
       test_tp, "redpanda.remote.read", "true", describe_resp);
 }
+
```

Comment:
I may have just missed it, but is there a test case for 

- SI enabled for some topic already
- non-incremental alter config that changes some other property

Reply:
Done